### PR TITLE
add Modal tests

### PR DESCRIPTION
**Pull request by kshen0**
Adds basic tests for the `Modal` component

cc @rgarcia

### CODE REVIEW

File: test/Modal/Modal_test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  it('calls closeModal when the background is clicked', function() {
+    var stub = sinon.stub()
+    this.renderModal(<Modal title="Title" content={<p>Content</p>} closeModal={stub} />);
+    var background = this.modal.props.children[0];
+    background.props.onClick()
+    this.renderModal();
+    assert(stub.calledOnce);
+  });
+});
```
Comment (rgarcia) [inline test/Modal/Modal_test.js]: consider ES6!

**Comment by rgarcia** [comment]
:heart:

File: test/Modal/Modal_test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+var assert = require('assert');
```
Comment (jacobsimon) [inline test/Modal/Modal_test.js]: Do you think we'll need more than 1 test file per component? If not we probably don't need a separate directory for each one

**Comment by jacobsimon** [comment]
Awesome! We might want to add a [.npmignore](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) to exclude test files from the package

File: test/Modal/Modal_test.js (hunk 2 above)
Comment (kshen0) [inline test/Modal/Modal_test.js]: abc6ec7

**Comment by kshen0** [comment]
`.npmignore` added in b5a5a69

**Comment by jacobsimon** [comment]
lgtm

**Pull request merged**